registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -96,6 +96,7 @@
     "sha256:a316a24bd7d2e61cae26e546e23889ef85e42d6d4e24bdb0a885ec9e3194b78a": ["v1.11.0-beta.0"]
     "sha256:d07762fd9a7e9f3e2cffa229c222c5d75d2b9185494dbd6a2962cb51dcafa370": ["v1.11.0-beta.1"]
     "sha256:11a86e5056384f15b544ecd77f551ce9faabb06800b34513007da5d2b75cc085": ["v1.11.0-beta.2"]
+    "sha256:d115f04597212b26bf2f17e9b056821f86a9cc5ad5bd4dcb7bd27c5586df7f35": ["v1.11.0-rc.0"]
```

Comment:
SHA `d115f0459721` matches cluster-api-controller v1.11.0-rc.0 in the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/cluster-api-controller@sha256:d115f04597212b26bf2f17e9b056821f86a9cc5ad5bd4dcb7bd27c5586df7f35/details?inv=1&invt=Ab4ryw).